hudi-timeline-service/pom.xml

```diff
@@ -87,6 +87,12 @@
       <artifactId>kryo-shaded</artifactId>
     </dependency>
 
+    <dependency>
+      <groupId>org.eclipse.jetty</groupId>
+      <artifactId>jetty-util</artifactId>
+      <version>${jetty.version}</version>
```

Comment:
How could the confclit happen, are multiple jars of jetty existing on the dependency tree?

Reply:
Problem recurrence command：
```
mvn clean package -DskipTests -Dspark3.1 -Dscala-2.12 -Dflink1.17 -Dhive.version=3.1.2 -Pflink-bundle-shade-hive3 -Dhadoop.version=3.3.0 -Dcheckstyle.skip=true -Drat.skip=true  -pl packaging/hudi-flink-bundle -am 
```

There is only one version of jetty jar on the dependency tree of `hudi-timeline-service`.

Reply:
Can you explan how the change fixes the problem, was wrong jetty version was used before?

Reply:
Did not find the root cause, maybe the wrong version was loaded first.

Reply:
Let's make it clear first.

Reply:
@danny0405  I ran into the same problem. After some analysis, the time-line-server did not directly reference the jetty-util jar, but referenced hadoop dependencies. Hadoop version 3.3.0 refers to version 9.4.20.v20190813 of jetty-util (https://github.com/eclipse/jetty.project/blob/jetty-9.4.20.v20190813/jetty-util/src/main/java/org/eclipse/jetty/util/thread/ScheduledExecutorScheduler.java), the constructor of this version does not contain ScheduledExecutorScheduler(@Name("name") String name, @Name("daemon") boolean daemon, @Name(" threads") int threads), so when compiling time-line-server, the jetty-util 9.4.20.v20190813 is referenced instead of the 9.4.48.v20220622 version in the parent pom.

Reply:
Thanks for the clarification ~

Reply:
Thanks for ur help!